src/parser/mmtf-parser.ts

```diff
@@ -122,7 +123,7 @@ class MmtfParser extends StructureParser {
 
     const gChainIndex = new Uint32Array(numGroups)
     const gAtomOffset = new Uint32Array(numGroups)
-    const gAtomCount = new Uint16Array(numGroups)
+    const gAtomCount = new Uint32Array(numGroups)
```

Comment:
This change is probably wrong. gAtomCount contains the number of atoms in a residue and 65k seems more than enough

Reply:
This is a consequence of this PR #825 . I suppose that this can happen when the file contains only a single residue, that is no residues are defined.

In this PR, because atomStore is properly typed in the StructureParsers, an error is raised by the previous change, which is why this change had been made.